test/lib/range_parser_test.rb

```diff
@@ -33,4 +33,26 @@ class RangeParserTest < ActiveSupport::TestCase
     assert_equal (0.1..0.33), RangeParser.parse('0.1   ,   0.33')
   end
 
+  test 'parses garbage format from JSONAPI' do
```

Comment:
Hah. This happens because JSONAPI is smart (:sushi:) enough to split comma-separated params. Maybe the client should send filters like `filter[commsf]=min,max` instead of putting braces around it.